SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -166,6 +170,19 @@ public SingularityRequest checkSingularityRequest(SingularityRequest request, Op
     return request.toBuilder().setQuartzSchedule(Optional.fromNullable(quartzSchedule)).build();
   }
 
+  public SingularityWebhook checkSingularityWebhook(SingularityWebhook webhook) {
+    checkNotNull(webhook, "Webhook is null");
+    checkNotNull(webhook.getUri(), "URI is null");
+
+    try {
+      new URI(webhook.getUri());
+    } catch (URISyntaxException e) {
+      checkBadRequest(false, "Invalid URI provided");
```

Comment:
Might as well do `WebExceptions.badRequest("Invalid URI provided")` here for clarity, since the condition argument is always false